packages/hardhat-solhint/src/index.ts

```diff
@@ -127,4 +127,15 @@ task("check", async (_, { run }, runSuper) => {
   const reports = await run("hardhat-solhint:run-solhint");
 
   printReport(reports);
+
+  const errorsCount = reports.reduce(
+    (acc: number, i: { errorCount: number }) => {
+      return acc + i.errorCount;
+    },
+    0
+  );
+
+  if (errorsCount > 0) {
```

Comment:
I think this is legitimate. My expectation is for a set of linters being run is to fail fast. Our plugin system does not encode patterns for parallelism, so we should assume linear running of check tasks.
Basically I think we should exit out after this task if there are errors.

Reply:
Thanks! I was just about to ask on the dev channel. Should I follow up on it there anyway or do you think it's OK for us to proceed?